test/utils.c

```diff
@@ -256,6 +256,138 @@ static void fakeroot_test(void)
 	g_assert_true(res);
 }
 
+static void semver_parse_test(void)
+{
+	g_autofree gchar *version_string;
+	g_autofree guint version[3];
+	g_autofree gchar *pre_release;
+	g_autofree gchar *build;
+
+	version_string = g_strdup("1.2.3");
```

Comment:
These leak memory and will cause address sanitizer errors. Take a look at our `replace_strdup` helper and/or use const string literals if modification is not needed.

Reply:
changed to replace_strdup with v3